File: java/arrays/reverse_array.java
import java.util.Scanner;

public class reverse_array {
    public static void main(String[] args) {
        // to reverse an array
        Scanner sc = new Scanner(System.in);
        int n=sc.nextInt();
        int arr[]= new int[n];
        int len=arr.length;
        for(int i=0;i<len;i++){
            arr[i]=sc.nextInt();
        }
        for(int i=len;i>=0;i--){
            System.out.println("Reverse array"+ " "+arr);
        }

        }
    }
    

File: interact os with python/Python_problem/arrays/reverse_array.py
def Reverse_array(arr):
    lenght=len(arr)
    
    print("Orginal array")
    for i in range(0,lenght):
        print(arr[i])
        
    print("Reverse array")
    
    for i in range(lenght-1,-1,-1):
        print(arr[i])
        
arr=[1,2,3,4,5,6]
print(Reverse_array(arr))            